### ISSUE TITLE
Cannot Find Module "browserify"

### ISSUE THREAD

**Issue by lukad03** [presidential-innovation-fellows/apps-gov#136]
There's an issue with the `jekyll_pages_api_search` plugin that's prohibiting future development on apps.gov - https://github.com/18F/jekyll_pages_api_search/issues/38

**Comment by stroupaloop** [comment]
possibly a @mblad fix?

**Comment by stroupaloop** [comment]
possibly a @mbland fix?

**Comment by mbland** [comment]
@lukad03 What happens if you do a `./go init` (which invokes`npm install`) and try again?

**Comment by mbland** [comment]
(Wow, just noticed this was filed a while ago...)

**Comment by stroupaloop** [comment]
Indeed, just noticed this as well, thought I'd go through and check out the Issues page ...
